qfieldsync/gui/project_configuration_widget.py

```diff
@@ -62,9 +62,11 @@ def __init__(self, parent=None):
         self.areaOfInterestExtentWidget.setToolTip(
             self.tr("Leave null to use the current project zoom extent.")
         )
-        self.areaOfInterestExtentWidget.setMapCanvas(iface.mapCanvas())
         self.areaOfInterestExtentWidget.setNullValueAllowed(True)
 
+        if iface:
```

Comment:
How can this be run without iface again?

Reply:
I guess when run from commandline, iface is `None`, it failed the previous job

Reply:
is the `gui` lib used from `cli`?

Reply:
no, none of them is using the gui:
```
import os
import shutil
import tempfile

from qgis.core import Qgis, QgsOfflineEditing, QgsProject
from qgis.testing import start_app, unittest
from qgis.testing.mocked import get_iface

from qfieldsync.libqfieldsync import OfflineConverter
from qfieldsync.tests.utilities import test_data_folder

start_app()
```